### PR TITLE
Update Sponsors and Sponsorhip information

### PR DESCRIPTION
**Pull request by pronskiy**

### CODE REVIEW

File: source/_layouts/sponsors.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                <img class="h-32 rounded w-full object-contain object-center"
+                     src="{{ sponsor.image }}" alt="sponsor.name"/>
+            </div>
+            <div class="overflow-hidden overflow-ellipsis whitespace-nowrap text-gray-900 font-medium mb-4 text-center">
+                <a class="no-underline text-inherit" href="{{ sponsor.website }}">{{ sponsor.name }}</a>
+            </div>
+        </a>
+    </div>
+    {% endfor %}
+</div>
```
Comment (saundefined) [inline source/_layouts/sponsors.html:19]:
```suggestion
{% if sponsors['Platinum']|length > 0 %}
<h2 id="sponsors_platinum">Platinum Sponsors</h2>
<div class="grid grid-cols-4 gap-4 mb-8">
    {% for sponsor in sponsors['Platinum'] %}
    <div>
        <a href="{{ sponsor.website }}"
           class="block no-underline" target="_blank" rel="noopener noreferrer">
            <div class="mb-2 p-2">
                <img class="h-32 rounded w-full object-contain object-center"
                     src="{{ sponsor.image }}" alt="sponsor.name"/>
            </div>
            <div class="overflow-hidden overflow-ellipsis whitespace-nowrap text-gray-900 font-medium mb-4 text-center">
                <a class="no-underline text-inherit" href="{{ sponsor.website }}">{{ sponsor.name }}</a>
            </div>
        </a>
    </div>
    {% endfor %}
</div>
{% endif %}
```

File: source/_layouts/sponsors.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                <img class="h-24 rounded w-full object-contain object-center"
+                     src="{{ sponsor.image }}" alt="sponsor.name"/>
+            </div>
+            <div class="overflow-hidden overflow-ellipsis whitespace-nowrap text-gray-900 font-medium mb-4 text-center">
+                <a class="no-underline text-inherit" href="{{ sponsor.website }}">{{ sponsor.name }}</a>
+            </div>
+        </a>
+    </div>
+    {% endfor %}
+</div>
```
Comment (saundefined) [inline source/_layouts/sponsors.html:36]:
```suggestion
{% if sponsors['Gold']|length > 0 %}
<h2 id="sponsors_gold">Gold Sponsors</h2>
<div class="grid grid-cols-6 gap-4 text-base mb-8">
    {% for sponsor in sponsors['Gold'] %}
    <div>
        <a href="{{ sponsor.website }}"
           class="block no-underline" target="_blank" rel="noopener noreferrer">
            <div class="mb-2 p-2">
                <img class="h-24 rounded w-full object-contain object-center"
                     src="{{ sponsor.image }}" alt="sponsor.name"/>
            </div>
            <div class="overflow-hidden overflow-ellipsis whitespace-nowrap text-gray-900 font-medium mb-4 text-center">
                <a class="no-underline text-inherit" href="{{ sponsor.website }}">{{ sponsor.name }}</a>
            </div>
        </a>
    </div>
    {% endfor %}
</div>
{% endif %}
```

File: source/_layouts/sponsors.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
```
Comment (saundefined) [inline source/_layouts/sponsors.html:54]:
```suggestion
{% if sponsors['Silver']|length > 0 %}
<h2 id="sponsors_silver">Silver Sponsors</h2>
<div class="grid grid-cols-6 gap-4 text-sm  mb-8">
    {% for sponsor in sponsors['Silver'] %}
    <div>
        <a href="{{ sponsor.website }}"
           class="block no-underline" target="_blank" rel="noopener noreferrer">
            <div class="mb-2 p-2">
                <img class="h-24 rounded w-full object-contain object-center"
                     src="{{ sponsor.image }}" alt="sponsor.name"/>
            </div>
            <div class="overflow-hidden overflow-ellipsis whitespace-nowrap text-gray-900 font-medium mb-4 text-center">
                <a class="no-underline text-inherit" href="{{ sponsor.website }}">{{ sponsor.name }}</a>
            </div>
        </a>
    </div>
    {% endfor %}
</div>
{% endif %}
```

**Review by saundefined — APPROVED** [review]
This looks great, thank you!

Maybe add a check for sponsors in the level?
Otherwise, silver sponsors have a heading but no companies (I guess temporarily, but still) 😀

File: source/_layouts/sponsors.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+{% endif %}
+
+{% if sponsors['Silver']|length > 0 %}
+<h2 id="sponsors_silver">Silver Sponsors</h2>
+<div class="grid grid-cols-6 gap-4 text-base  mb-8">
```
Comment (saundefined) [inline source/_layouts/sponsors.html:44]:
```suggestion
<div class="grid grid-cols-2 lg:grid-cols-6 gap-4 text-base mb-8">
```

File: source/_layouts/sponsors.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            </div>
+        </a>
+    </div>
+    {% endfor %}
+</div>
+{% endif %}
+
+{% if sponsors['Gold']|length > 0 %}
+<h2 id="sponsors_gold">Gold Sponsors</h2>
+<div class="grid grid-cols-5 gap-4 text-base mb-8">
```
Comment (saundefined) [inline source/_layouts/sponsors.html:24]:
```suggestion
<div class="grid grid-cols-2 lg:grid-cols-5 gap-4 text-base mb-8">
```

File: source/_layouts/sponsors.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
+{% set sponsors = fetch_sponsors() %}
+{% if sponsors['Platinum']|length > 0 %}
+<h2 id="sponsors_platinum">Platinum Sponsors</h2>
+<div class="grid grid-cols-4 gap-4 mb-8">
```
Comment (saundefined) [inline source/_layouts/sponsors.html:4]:
```suggestion
<div class="grid grid-cols-2 lg:grid-cols-4 gap-4 mb-8">
```

File: source/_layouts/sponsors.html [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+            </div>
+        </a>
+    </div>
+    {% endfor %}
+</div>
+{% endif %}
+
+{% if sponsors['Supporters']|length > 0 %}
+<h3 id="supporters">Supporters</h3>
+<div class="grid grid-cols-8 gap-4 text-base mb-8">
```
Comment (saundefined) [inline source/_layouts/sponsors.html:64]:
```suggestion
<div class="grid grid-cols-2 lg:grid-cols-6 gap-4 text-base mb-8">
```

**Review by saundefined — COMMENTED** [review]
Nits for mobile

File: source/_layouts/sponsors.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
+{% set sponsors = fetch_sponsors() %}
+{% if sponsors['Platinum']|length > 0 %}
+<h2 id="sponsors_platinum">Platinum Sponsors</h2>
+<div class="grid grid-cols-2 lg:grid-cols-4 gap-4 mb-8">
+    {% for sponsor in sponsors['Platinum'] %}
+    <div>
```
Comment (lwlwilliam) [inline source/_layouts/sponsors.html:6]:
```suggestion
    <div title="{{ sponsor.name }}">
```

File: source/_layouts/sponsors.html [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+    </div>
+    {% endfor %}
+</div>
+{% endif %}
+
+{% if sponsors['Gold']|length > 0 %}
+<h2 id="sponsors_gold">Gold Sponsors</h2>
+<div class="grid grid-cols-2 lg:grid-cols-5 gap-4 text-base mb-8">
+    {% for sponsor in sponsors['Gold'] %}
+    <div>
```
Comment (lwlwilliam) [inline source/_layouts/sponsors.html:26]:
```suggestion
    <div title="{{ sponsor.name }}">
```

File: source/_layouts/sponsors.html [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+    </div>
+    {% endfor %}
+</div>
+{% endif %}
+
+{% if sponsors['Silver']|length > 0 %}
+<h2 id="sponsors_silver">Silver Sponsors</h2>
+<div class="grid grid-cols-2 lg:grid-cols-6 gap-4 text-base mb-8">
+    {% for sponsor in sponsors['Silver'] | filter(sponsor => sponsor.name) %}
+    <div>
```
Comment (lwlwilliam) [inline source/_layouts/sponsors.html:46]:
```suggestion
    <div title="{{ sponsor.name }}">
```

**Review by lwlwilliam — CHANGES_REQUESTED** [review]
Some sponsors may have names that are too long to fully display.

File: source/_layouts/sponsors.html [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+{% if sponsors['Platinum']|length > 0 %}
+<h2 id="sponsors_platinum">Platinum Sponsors</h2>
+<div class="grid grid-cols-2 lg:grid-cols-4 gap-4 mb-8">
+    {% for sponsor in sponsors['Platinum'] %}
+    <div title="{{ sponsor.name }}">
+        <a href="{{ sponsor.website }}"
+           class="block no-underline" target="_blank" rel="noopener noreferrer">
+            <div class="mb-2 p-2">
+                <img class="h-32 rounded w-full object-contain object-center"
+                     src="{{ sponsor.image }}" alt="sponsor.name"/>
```
Comment (lwlwilliam) [inline source/_layouts/sponsors.html:11]:
```suggestion
                     src="{{ sponsor.image }}" alt="{{ sponsor.name }}"/>
```

File: source/_layouts/sponsors.html [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+{% if sponsors['Gold']|length > 0 %}
+<h2 id="sponsors_gold">Gold Sponsors</h2>
+<div class="grid grid-cols-2 lg:grid-cols-5 gap-4 text-base mb-8">
+    {% for sponsor in sponsors['Gold'] %}
+    <div title="{{ sponsor.name }}">
+        <a href="{{ sponsor.website }}"
+           class="block no-underline" target="_blank" rel="noopener noreferrer">
+            <div class="mb-2 p-2">
+                <img class="h-24 rounded w-full object-contain object-center"
+                     src="{{ sponsor.image }}" alt="sponsor.name"/>
```
Comment (lwlwilliam) [inline source/_layouts/sponsors.html:31]:
```suggestion
                     src="{{ sponsor.image }}" alt="{{ sponsor.name }}"/>
```

File: source/_layouts/sponsors.html [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+{% if sponsors['Silver']|length > 0 %}
+<h2 id="sponsors_silver">Silver Sponsors</h2>
+<div class="grid grid-cols-2 lg:grid-cols-6 gap-4 text-base mb-8">
+    {% for sponsor in sponsors['Silver'] | filter(sponsor => sponsor.name) %}
+    <div title="{{ sponsor.name }}">
+        <a href="{{ sponsor.website }}"
+           class="block no-underline" target="_blank" rel="noopener noreferrer">
+            <div class="mb-2 p-2">
+                <img class="h-20 rounded w-full object-contain object-center"
+                     src="{{ sponsor.image }}" alt="sponsor.name"/>
```
Comment (lwlwilliam) [inline source/_layouts/sponsors.html:51]:
```suggestion
                     src="{{ sponsor.image }}" alt="{{ sponsor.name }}"/>
```

**Review by lwlwilliam — CHANGES_REQUESTED** [review]

**Pull request merged**